freq/static/freq/js/buttons.js

```diff
@@ -0,0 +1,149 @@
+/*global $:false, jQuery:false, moment:false, ko:false, fillChart:false, */
+
+
+function datePickerValue() {
+    return {
+        start: $('#date_picker_start').val(),
+        end: $('#date_picker_end').val()
+    };
+}
+
+
+function spinnerValue(i) {
+    return function(){
```

Comment:
functies die functies returnen? Ik weet niet veel van javascript af, dus je mag me uitleggen wat dit doet en waarom dit handig is :-)